contrib/dhcp_agent/setup.py

```diff
@@ -0,0 +1,19 @@
+# -*- encoding: utf-8 -*-
+from setuptools import setup
+
+setup(
+    name='dhcp_agent',
+    version='0.0.1',
```

Comment:
For the initial version I'd rather use `0.1.0` - according to semver, `0.0.1` means "bugfix over version `0.0.0`" :wink: